### DISCUSSION TITLE
Interpretation of Confusion Matrix in Yolov5.

### DISCUSSION THREAD

**Discussion by rabiyaabbasi** [ultralytics/yolov5#7413] [Q&A]
I am new to YoloV5. I have trained my data and got the confusion matrix. Somehow I am unable to interpret the confusion matrix obtained after training. Particularly for class (Fungal-Powdery-Mildew and Viral-Lettuce-Mosaic). For Fungal-powdery-mildew I understand 67% of samples are predicted correctly whereas 33% are confused with Bacterial_leaf-Spots but then what does the last column with background-FP=0.50 interpret? Likewise for class viral-lettuce-mosaic, I understand that 100% of samples are predicted correctly but then what's the meaning of background-FP=0.50. I am totally lost. Any help will be welcome
![Picture1](https://user-images.githubusercontent.com/54917838/163265301-67db6fef-d288-476b-9e7d-6acab1e1040d.png)

**Comment by glenn-jocher** [comment]
@rabiyaabbasi last column displays what classes FPs are.

**Reply by shikhakalinga1** [reply]
please explain in detail

**Reply by egr42** [reply]
I think it means there were false positives from the background which shouldn't have been classified at all, 0.5 of these were classified as one class and 0.5 as the other. Likewise, background FN is probably objects which wern't classified as anything but should have been.

**Comment by abhisheksakhare1000** [comment]
> I am new to YoloV5. I have trained my data and got the confusion matrix. Somehow I am unable to interpret the confusion matrix obtained after training. Particularly for class (Fungal-Powdery-Mildew and Viral-Lettuce-Mosaic). For Fungal-powdery-mildew I understand 67% of samples are predicted correctly whereas 33% are confused with Bacterial_leaf-Spots but then what does the last column with background-FP=0.50 interpret? Likewise for class viral-lettuce-mosaic, I understand that 100% of samples are predicted correctly but then what's the meaning of background-FP=0.50. I am totally lost. Any help will be welcome ![Picture1](https://user-images.githubusercontent.com/54917838/163265301-67db6fef-d288-476b-9e7d-6acab1e1040d.png)

can you please explain me these below confusion matrix
![confusion_matrix](https://github.com/ultralytics/yolov5/assets/96978179/8bd83e7c-29ac-4724-8e09-c915d86c8ff4)
